src/user/user.directive.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+
+angular.module('oscra-ui.user').directive('userinfo', function(){
+    return {
+        restrict: 'E',
+        scope: {
+            tuser: '=suser'
+        },
+        template: require('./directiveTemplate/userAllInfo.html'),
+        link: function($scope){}
```

Comment:
if you don't use link, then you don't  need to implement a 'directive'. instead, use a 'component'